Gruntfile.js

```diff
@@ -564,11 +630,10 @@ module.exports = (grunt) => {
     // and then we build the main bundle since it depends on manifests.json
     await Promise.all([buildContracts.run(), buildContractsSlim.run()])
       .then(() => {
-        return genManifestsAndDeploy(distContracts, packageJSON.contractsVersion)
-      })
-      .then(() => {
-        return Promise.all([buildMain.run(), buildServiceWorkers.run()])
+        const dest = !this.flags.deploy ? undefined : `./${distDir}/data`
```

Comment:
Why not use `${distDir}/data` directly? It works the same for relative paths, but makes it easier to use absoulute paths.